src/ci/BitbucketPipelinesAdapter.js

```diff
@@ -10,7 +10,7 @@ type Config = {
   botPassword: string
 };
 
-const BitbucketPipelinesAdapter: CIAdapter = async (config: Config) => {
+const BitbucketPipelinesAdapter: CIAdapter = (config: Config) => {
```

Comment:
I will 100% look at adding this all back in/fixing the flow types later, Promise 👌